src/transformers/models/marian/convert_marian_to_pytorch.py

```diff
@@ -701,7 +701,12 @@ def unzip(zip_path: str, dest_dir: str) -> None:
     """
     parser = argparse.ArgumentParser()
     # Required parameters
-    parser.add_argument("--src", type=str, help="path to marian model sub dir", default="en-de")
+    parser.add_argument(
+        "--src",
+        type=str,
+        help="path to marian model sub dir. yaml.load will be used to load the configuration file, please be weary of which file you're loading.",
```

Comment:
turbo nit (although i realise these reports are tiresome :))
```suggestion
        help="path to marian model sub dir. yaml.load will be used to load the configuration file, please be wary of which file you're loading.",
```